src/benchmark.cc

```diff
@@ -753,6 +963,7 @@ void RunInThread(const benchmark::internal::Benchmark::Instance* b,
     total->bytes_processed += st.bytes_processed();
     total->items_processed += st.items_processed();
     total->complexity_n += st.complexity_length_n();
+    counters_total->Sum(st.Counters());
```

Comment:
Good suggestion. I'll move them there. (This happened because in my initial implementation there was no ThreadStats, then when I rolled back it was just quicker).